examples/plot_example_workflow.py

```diff
@@ -0,0 +1,151 @@
+ar = autoreject.AutoReject(n_interpolate=[1, 2, 3, 4], random_state=11,
+                           n_jobs=1, verbose='tqdm')
+ar.fit(epochs[:20])  # fit on the first 20 epochs to save time
+epochs_ar, reject_log = ar.transform(epochs, return_log=True)
+
+# visualize the dropped epochs
+epochs[reject_log.bad_epochs].plot(scalings=dict(eeg=1e-4))
+
+# %%
+# The data may be very valuable and the time for the experiment
+# limited and so we may want to take steps to reduce the number of
+# epochs dropped by first using other steps to preprocess the data.
+#
+# We will use a highpass filter first to remove slow drift that could
+# cause epochs to be dropped. As we can see in the plot, this reduced
+# the number of epochs marked as bad by autoreject substantially.
+#
+# When making this decision to filter the data, we do want to be careful
+# because filtering can spread sharp, high-frequency transients and
+# distort the phase of the signal. Most evoked response potential
+# analyses use filtering since the interest is in the time series, but
+# if you are doing a frequency based analysis, filtering before the
+# Fourier transform could potentially be avoided by detrending instead.
+
+raw.filter(l_freq=1, h_freq=None)
+epochs = mne.Epochs(raw, events, tmin=0, baseline=None, preload=True)
+ar = autoreject.AutoReject(n_interpolate=[1, 2, 3, 4], random_state=11,
+                           n_jobs=1, verbose='tqdm')
+ar.fit(epochs[:20])  # fit on the first 20 epochs to save time
+epochs_ar, reject_log = ar.transform(epochs, return_log=True)
+
+# visualize the dropped epochs
+epochs[reject_log.bad_epochs].plot(scalings=dict(eeg=1e-4))
+
+# %%
+# Finally, we can apply independent components analysis (ICA) to remove
+# eyeblinks from the data. If our analysis were to be very dependent on
+# sensors at the front of the head or frequency components near the
+# frequency of eyeblink potentials or if we wanted to be extra careful
+# or if we had lots of data and didn't want to go through the trouble,
+# we could skip ICA and use the previous result. However, ICA can increase
+# the amount of usable data and is one of the most commonly used methods
+# in EEG analyses.
+#
+# We can see in the plots below that ICA effectively removed eyeblink
+# artifact, and, a bit counterintuitively, in doing so caused a few more
+# epochs to be dropped. This is because the threshold was no longer biased
+# upwards by the eyeblinks and so was more stringent.
+#
+# These are the basic steps for a workflow with decisions that must be
+# made based on what the data is being used for. Following this may help
+# you optimize your use of ``autoreject`` in preprocessing.
+
+ica = mne.preprocessing.ICA(random_state=99)
+reject = autoreject.get_rejection_threshold(epochs)
+ica.fit(epochs, reject=reject)
+# get the rejected components
+
+# plot with and without eyeblink component
+ica.plot_overlay(epochs.average(), exclude=ica.exclude)
```

Comment:
I am a little confused, how do you exclude eye blink components ? I usually plot the components and see the erp and then choose those components to be excluded.

Reply:
Based on the rejection threshold